org.aposin.licensescout.core/src/test/java/org/aposin/licensescout/core/test/util/TestUtil.java

```diff
@@ -15,15 +15,24 @@
  */
 package org.aposin.licensescout.core.test.util;
 
+import java.io.File;
 import java.util.logging.Logger;
 
 import org.aposin.licensescout.configuration.RunParameters;
+import org.aposin.licensescout.license.LicenseStoreData;
 import org.aposin.licensescout.util.ILFLog;
 import org.aposin.licensescout.util.JavaUtilLog;
 import org.aposin.licensescout.util.NullLog;
 
+//import java.util.logging.Logger;
+//
+//import org.aposin.licensescout.configuration.RunParameters;
+//import org.aposin.licensescout.util.ILFLog;
+//import org.aposin.licensescout.util.JavaUtilLog;
+//import org.aposin.licensescout.util.NullLog;
```

Comment:
Is there any reason to keep those commented imports?

Reply:
No - left over. I deleted them.